LiteCore/Support/Logging.hh

```diff
@@ -166,6 +166,7 @@ extern LogDomain DBLog, QueryLog, SyncLog, &ActorLog;
 #define Log(FMT, ...)                   LogToAt(kC4Cpp_DefaultLog, Info,    FMT, ##__VA_ARGS__)
 #define Warn(FMT, ...)                  LogToAt(kC4Cpp_DefaultLog, Warning, FMT, ##__VA_ARGS__)
 #define WarnError(FMT, ...)             LogToAt(kC4Cpp_DefaultLog, Error,   FMT, ##__VA_ARGS__)
+#define WriteVerbose(FMT, ...)          LogToAt(kC4Cpp_DefaultLog, Verbose, FMT, ##__VA_ARGS__)
 #define WriteDebug(FMT, ...)            LogToAt(kC4Cpp_DefaultLog, Debug,   FMT, ##__VA_ARGS__)
```

Comment:
We already have `LogVerbose` and `LogDebug`. Why are these new ones needed?

Reply:
WriteDebug was already there, I just made verbose available.  These write without specifying a log domain, as opposed to the others which accept the domain as their first argument.